machine.go

```diff
@@ -95,8 +112,9 @@ func InMachine() (ret bool) {
 	return
 }
 
+// Checks whether at least one backend is supported
 func Supported() bool {
-	_, err := os.Stat("/dev/kvm")
+	_, err := NewMachineWithBackend("auto")
```

Comment:
This shows the awkward machine/backend relationship ... but lets refactor that later :)